docs/BUILD.md

```diff
@@ -0,0 +1,69 @@
+# Building Kubicorn
+
+## Make
+
+First off you will need to have a working Golang-1.8 development environment. 
+Make sure you can use the `go` command and that your `GOPATH` environment variable is set.
+Als a alternative you can have a look at the [Docker build script](#docker), you wont need `go` localy for this.    
+
+### Environment
+
+Set `GOPATH` in your bash profile:
+```bash
+$ export GOPATH=/Users/<your user>/go
+$ export PATH=$GOPATH/bin:$PATH
+```
+You can add these two lines to `~/.bash_profil` to make sure these variables are always.
+
+If you have this you should be able to run the following command:
+
+```bash
+$ go get github.com/kris-nova/kubicorn
+```
+
+### Building
+Now you can run `make` from the src directory of kubicorn:
+
+```bash
+$ cd $GOPATH/src/github.com/kris-nova/kubicorn/
+$ make
+```
+The kubicorn binary will get built and placed under `$GOPATH/bin` and `$GOPATH/src/github.com/kris-nova/kubicorn/`. 
+Run kubicorn -h to get the nice Unicorn:
+
+```bash
+$ kubicorn -h
+```
+
+### Other branches or your fork
```

Comment:
I think its a bit overkill to add how to fork a repo. But adding a link (https://help.github.com/articles/fork-a-repo/) is maybe a good alternative.